pylmod/membership.py

```diff
@@ -18,7 +19,106 @@ def __init__(
             self,
             cert,
             urlbase='https://learning-modules.mit.edu:8443/',
+            cuuid=None
     ):
         super(Membership, self).__init__(cert, urlbase)
         # Add service base
         self.urlbase += 'service/membership/'
+        if cuuid is not None:
+            self.course_id = self.get_course_id(cuuid)
+
+    def get_staff(self, course_id='', simple=False):
+        """Get the staff roster for a course.
+
+        Get a list of staff members for a given course,
+        specified by a course id.
+
+        Args:
+            course_id (int): unique identifier for course, i.e. ``2314``
+
+        Raises:
+            requests.RequestException: Exception connection error
+            ValueError: Unable to decode response content
+
+        Returns:
+            list: list of dictionaries containing staff data
+
+            An example return value is:
+
+            .. code-block:: python
+
+                {
+                    u'response':
+                    {
+                        u'docs':
+                        [
+                            {
+                                u'displayName': u'Huey Duck',
+                                u'role': u'TA',
+                                u'sortableDisplayName': u'Duck, Huey'
+                            },
+                            {
+                                u'displayName': u'Louie Duck',
+                                u'role': u'CourseAdmin',
+                                u'sortableDisplayName': u'Duck, Louie'
+                            },
+                            {
+                                u'displayName': u'Benjamin Franklin',
+                                u'role': u'CourseAdmin',
+                                u'sortableDisplayName': u'Franklin, Benjamin'
+                            },
+                            {
+                                u'displayName': u'George Washington',
+                                u'role': u'Instructor',
+                                u'sortableDisplayName': u'Washington, George'
+                            },
+                        ],
+                        u'numFound': 16,
+                        u'start': 0
+                    },
+                    u'responseHeader':
+                    {
+                        u'QTime': 0,
+                        u'params':
+                        {
+                            u'wt': u'json'
+                        },
+                        u'qtime': 0,
+                        u'status': 0
+                    }
+                }
+    """
+        staff_data = self.get(
+            'courseguide/course/{courseId}/staff'.format(
+                courseId=course_id or self.course_id
+            ),
+            params=None
+        )
+
+        if simple:
+            return staff_data
+        return staff_data
+
+    def get_course_id(self, course_uuid=''):
```

Comment:
nit: I think it makes more sense to have this next to `__init__` since it is called there.